bundles/org.openhab.binding.plugwise/README.md

```diff
@@ -23,7 +24,6 @@ The binding supports the following Plugwise devices:
 ## Discovery
 
 Automatic device discovery runs every 3 minutes which can be sped up by starting a manual discovery.
```

Comment:
Is there another discovery service other than the one you deleted?

Reply:
Yes the `PlugwiseThingDiscoveryService` :-)